### DISCUSSION TITLE
Harness.io Integration

### DISCUSSION THREAD

**Discussion by Makisuo** [Infisical/infisical#5250] [Feature Request]
### Feature description
A integration for the popular CI/CD tool harness.io

### Why would it be useful?
Harness is a great tool for everything around CI/CD and GitOPS and would greatly benefit of a infisical integration

**Comment by maidul98** [comment]
I'm unfamiliar with Harness.io but does it also work in the same way github actions does with workflow files?

**Comment by Makisuo** [comment]
@maidul98 its very similar to things like CircleCI with an additional focus on GitOPS and other features like feature flags

**Comment by dangtony98** [comment]
We'll slot this integration in soon :)

**Comment by Aashish-Upadhyay-101** [comment]
I will be happy to do this integration. 😄

**Comment by Aashish-Upadhyay-101** [comment]
I think I might need some help!

**Comment by dangtony98** [comment]
No worries @Aashish-Upadhyay-101. We can circle back on this integration as well!

**Comment by afrieirham** [comment]
Would love to take this one. @dangtony98

**Comment by dangtony98** [comment]
@afrieirham Assigned!

**Comment by dangtony98** [comment]
Hey @afrieirham!

Are you still working on this issue? — I saw there's an unfinished draft PR.

**Comment by afrieirham** [comment]
Hey @dangtony98, as I remember, there was 1 issue with passing the `organizationId` from `backend/src/integrations/apps.ts` to `backend/src/integrations/sync.ts`.

I pushed the code if you want to take it from there.

**Comment by atimapreandrew** [comment]
@dangtony98 

Please assign this to me, would love to complete the integration.

**Comment by kisharnath** [comment]
Hey @atimapreandrew  are you working on this issue?
